src/test/pybind/test_rados.py

```diff
@@ -578,6 +578,26 @@ def test_clear_omap(self):
             self.ioctx.operate_read_op(read_op, "hw")
             eq(list(iter), [])
 
+    def test_remove_omap_ramge2(self):
+        keys = ("1", "2", "3", "4")
+        values = (b"a", b"bb", b"ccc", b"dddd")
+        with WriteOpCtx() as write_op:
+            self.ioctx.set_omap(write_op, keys, values)
+            self.ioctx.operate_write_op(write_op, "test_obj")
+        with ReadOpCtx() as read_op:
+            iter, ret = self.ioctx.get_omap_vals_by_keys(read_op,("1","2","3","4"))
+            eq(ret, 0)
+            self.ioctx.operate_read_op(read_op, "test_obj")
+            eq(list(iter), [("1", b"a"), ("2", b"bb"), ("3", b"ccc"),("4", b"dddd")])
```

Comment:
please use `zip(keys, values)`.

Reply:
thank you very much for your kindly advice, updated.